backend/app/services/collections/create_collection.py

```
@@ -281,37 +321,42 @@ def execute_job(
                     webhook_secret=webhook_secret,
                 )
 
+        except (Timeout, SoftTimeLimitExceeded) as err:
+            timeout_err = TimeoutError(f"Task exceeded soft time limit")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)